code/modules/economy/economy_misc.dm

```diff
@@ -113,7 +113,7 @@ var/global/economy_init = 0
 		next_account_number = rand(111111, 999999)
 
 		station_account = new()
-		station_account.owner_name = "[station_name()] Station Account"
+		station_account.owner_name = "[CONFIG_GET(string/ship_name)] Station Account"
```

Comment:
This one will be wrong no matter what you do, unless you do a larger refactor.
```suggestion
		station_account.owner_name = "[CONFIG_GET(string/ship_name)] Central Account"
```
That would at least lessen the issue.